Cargo.lock

```diff
@@ -12643,9 +12643,9 @@ dependencies = [
 
 [[package]]
 name = "roaring"
-version = "0.10.6"
+version = "0.10.10"
```

Comment:
are these 2 crate updates intentional in this PR ?

Reply:
No, but sometimes cargo is a mystery 👻

Reply:
The lock was probably not updated in a previous PR